runtime/autoload/remote/host.vim

```diff
@@ -130,7 +130,6 @@ endfunction
 function! remote#host#LoadRemotePluginsEvent(event, pattern) abort
   autocmd! nvim-rplugin
   call remote#host#LoadRemotePlugins()
-  execute 'silent doautocmd' a:event a:pattern
```

Comment:
@Streetwalrus instead of removing this, can you add `<nomodeline>`. Example:

https://github.com/junegunn/gv.vim/commit/2f5323e038088ab5ba1ffff25d416b2bae367360

And let me know if that fixes the issue?